WebdriverIO.md

```diff
@@ -0,0 +1,561 @@
+
+1. Find the Chrome browser version installed on your machine by going to `chrome://version/`. For example, in the screenshot below, the version number is `83.0.4103.61`.
+
+   ![Screenshot of Chrome version page.](https://user-images.githubusercontent.com/11008603/87473539-3c972880-c63f-11ea-9455-04edb0196731.png)
+
+2. Remove the last part of the version number from step 1 and append the result to URL `https://chromedriver.storage.googleapis.com/LATEST_RELEASE_`. For example, if your version number is `83.0.4103.61`, the URL will look like "https://chromedriver.storage.googleapis.com/LATEST_RELEASE_83.0.4103".
+
+3. Go to the URL from step 2 and copy the version number on the screen.
+
+4. The version number obtained in step 3 is the chromedriver version to be passed along to the script.
+
+If you see a failure due to the webdriver, please follow the instructions above to double check that the chromedriver version provided is in sync with the Chrome browser version installed on the machine.
+
+### Configure test sharding
+
+If you run all the E2E tests at once (i.e. if you don't specify a suite), the tests will be sharded across multiple Chrome browser instances. By default, the tests will use 3 shards (i.e. 3 browsers). If you do this, you should close background processes to maximize the compute resources available to the tests. You can configure the number of shards like this:
+
+```console
+python -m scripts.run_e2e_tests --sharding-instances=<number of shards>
+```
+
+You can disable sharding like this:
+
+```console
+python -m scripts.run_e2e_tests --sharding=false
+```
+
+Note that when we run tests on CI, we run one suite at a time, so there is no sharding.
+
+### Run a single end-to-end test
+
+To run just one test, change the "it" to "fit" for that test. Then when you run the tests, specify the suite containing your test.
+
+### Run end-to-end tests in production mode
+
+To run the end-to-end tests in production mode, use the `--prod_env` flag:
+
+```console
+python -m scripts.run_e2e_tests --prod_env
+```
+
+On CI, we run all the E2E tests in production mode to more closely mimic how the Oppia application behaves in production.
+
+## Write E2E tests
+
+We expect E2E tests to be written for any new feature that is released to the public.
+
+### Where to add the tests
+
+#### Interactions
+
+If you are just creating a new interaction and want to add end-to-end tests for it then you can follow the guidance given at [[Creating Interactions|Creating-Interactions]], though the [forms and objects](#forms-and-objects) section of this page may also be helpful.
+
+If you are adding functionality to an existing interaction, you can probably just add test cases to its `protractor.js` file. For example, the `AlgebraicExpressionInput` interaction's file is at [`oppia/extensions/interactions/AlgebraicExpressionInput/webdriverio.js`](https://github.com/oppia/oppia/blob/develop/extensions/interactions/AlgebraicExpressionInput/webdriverio.js).
```

Comment:
oppia/extensions/interactions/AlgebraicExpressionInput/webdriverio.js doesn't exist yet. Ditto elsewhere

Reply:
Removed the links for now, I will add the links when the files will be available.